src/libopensc/ctx.c

```diff
@@ -93,6 +93,8 @@ struct _sc_driver_entry {
 	void *(*func)(void);
 };
 
+
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/ctx.c -->